dnf/rpm/__init__.py

```diff
@@ -67,7 +67,7 @@ def detect_releasever(installroot):
         if is_py3bytes(releasever):
             releasever = str(releasever, "utf-8")
         return releasever
-    return None
+    return ''
```

Comment:
`None` seems to be the appropriate value to return when the detection failed. I think the fix should be elsewhere (in the tests?). I haven't tried to reproduce...

Reply:
Yes indeed, the fix could be in the test (I don't remember where yet) but at the very least, I wanted to have your feedback if I would have overlooked something. Originally, I had this idea to return `''` because 'rawhide' was a legit value so it was a first attempt.

Reply:
@lukash it seems that the fix is not needed anymore. I don't reproduce it so in case you would accept the fix for tests, would you like me to remove the detect releasever commit or you can cherry-pick simply the commit for tests?

Reply:
Not sure I entirely understand, but if you're saying the releasever fix is not needed anymore and you want to merge only the locales fix, please remove the releasever fix commit and I'll merge the other one :+1:

Reply:
Yes I've forced push to let only the locales fix.